src/main.rs

```diff
@@ -114,20 +119,15 @@ async fn execute_traces(start_block: u64, end_block: u64, should_run_known: bool
             let mut juno_manager = JunoManager::new(JunoBranch::Native).await.unwrap();
             let result = juno_manager
                 .simulate_block(block_number, SimulationStrategy::Binary)
-                .await;
-
-            match result {
-                // Note that this doesn't compare the reasons for failure or the result on a success
-                Ok(result) => {
-                    let successes = result.iter().filter(|result| result.is_correct()).count();
-                    info!(
-                        "Completed block {block_number} with {successes}/{} successes",
-                        result.len()
-                    );
-                    log_block_report(block_number, result);
-                }
-                Err(err) => error!("Error simulating transactions: {}", err),
-            }
+                .await
+                .unwrap();
+            // Note that this doesn't compare the reasons for failure or the result on a success
+            let successes = result.iter().filter(|result| result.is_correct()).count();
+            info!(
+                "Completed block {block_number} with {successes}/{} successes",
+                result.len()
+            );
+            log_block_report(block_number, result);
```

Comment:
Why are you unsafely unwrapping `result`?

Reply:
the same is here